src/Core/src/Stream/ResponseBodyStream.php

```diff
@@ -8,8 +8,11 @@
 
 /**
  * Stream a HTTP response body.
+ * This class is a NC layer for Http Client that does not supports toStream
+ * When calling `getChunks` or `getContentAsResource` it first, fully read download the Response Body in a blocking way.
```

Comment:
I was fine with the previous implementation. I just wanted a comment that said that we could t restart reading half way. 

I’m fine with leaving that “bug” since it is an edge case

Reply:
reverted. 
- added commetnts about the edge case
- added a check that throw an exception if user call getChunk several time before the end of the first